packages/flutter/lib/src/widgets/_html_element_view_web.dart

```diff
@@ -81,28 +85,34 @@ PlatformViewCreatedCallback? _createPlatformViewCallbackForElementCallback(
 
 class _HtmlElementViewController extends PlatformViewController {
   _HtmlElementViewController(
-    this.viewId,
-    this.viewType,
+    this.platformViewId,
+    this.platformViewType,
     this.creationParams,
+    this.flutterViewId,
   );
 
   @override
-  final int viewId;
+  int get viewId => platformViewId;
+  final int platformViewId;
 
   /// The unique identifier for the HTML view type to be embedded by this widget.
   ///
   /// A PlatformViewFactory for this type must have been registered.
-  final String viewType;
+  final String platformViewType;
 
   final dynamic creationParams;
 
+  /// The ID of the [FlutterView] that this platform view is rendererd into.
+  final int flutterViewId;
+
   bool _initialized = false;
 
   Future<void> _initialize() async {
     final Map<String, dynamic> args = <String, dynamic>{
-      'id': viewId,
-      'viewType': viewType,
+      'platformViewId': platformViewId,
+      'platformViewType': platformViewType,
```

Comment:
This one can't land until the `engine` change lands. Especially changing the name of the parameter :)

Reply:
(The engine change has landed)